classes/install/Upgrade.inc.php

```diff
@@ -429,6 +429,59 @@ function fixQueriesAssocTypes() {
 	}
 
 	/**
+	 * Convert comments to editors to queries.
+	 * @return boolean True indicates success.
+	 */
+	function convertCommentsToEditor() {
+		$submissionDao = Application::getSubmissionDAO();
+		$queryDao = DAORegistry::getDAO('QueryDAO');
+		$noteDao = DAORegistry::getDAO('NoteDAO');
+
+		$commentsResult = $submissionDao->retrieve(
+			'SELECT s.submission_id, s.comments_to_ed, s.date_submitted, sa.user_id
+			FROM submissions_tmp s
+			LEFT JOIN stage_assignments sa ON sa.submission_id = s.submission_id
+			LEFT JOIN user_groups ug ON ug.user_group_id = sa.user_group_id
+			WHERE s.comments_to_ed IS NOT NULL AND s.comments_to_ed != \'\' AND ug.role_id = 65536
```

Comment:
Should use constants if possible, and if not, add a comment to the magic number for grepping purposes.